combat-trainer.lic

```diff
@@ -2215,6 +2215,11 @@ class TrainerProcess
     when 'App Bundle'
       retreat
       bput("app my bundle quick", 'You scan', 'Appraise what', 'You cannot appraise', 'You can\'t appraise')
+    when /Summon (?<element>Aether|Air|Earth|Electricity|Fire|Water)/
```

Comment:
> * There is at least one easter-egg element available that is not one of the traditional six.
> * A painful ripping sensation erupts through your abdomen! You seem unharmed, but something terribly... wrong just happened. (Secret element)

I don't know anything about this secret element - should we support it?

Reply:
I don't know anything about that easter egg either -- whether it's just for fun or actually has an impact where you'd want to use it. If we add the word "Domain" at the end per your earlier comment, https://github.com/rpherbig/dr-scripts/pull/4941#discussion_r624704853, then this regex could simplify to `/Summon (?<element>.*) Domain/` to support any value the game allows as a summonable domain, even if we don't know what specifically that is.

Reply:
I kinda like the simplification, but then we'd need to also check for error text ("that isn't a valid domain" or such)